docs/authors.rst

```diff
@@ -0,0 +1,7 @@
+Authors
```

Comment:
Please also add `AUTHORS` etc which we use elsewhere... a reminder of the GNU packaging standards..